### PR TITLE
Il 53/login

### PR DESCRIPTION
**Pull request by isakk7**
# Description

Login was connected to the back with auth working, design the login

IMG_F7FE4BB42C05-1

# Checklist:

- [ ] Pull request owner assigned
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have pulled / merged main/develop into this branch

[IL-53]

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,8 +1,7 @@
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
 import { SafeAreaView } from 'react-native-safe-area-context';
 import MainStack from './navigation/MainStack';
 
+
```
Comment (JavierRuvalcaba) [inline App.js:4]: remove this line

File: components/Boton.jsx [hunk 2]
```diff
@@ -12,6 +12,7 @@ const Boton = (props) => {
         >
             <Text style= {styles.buttonText}>
                 {text}
+                Ingresa
```
Comment (JavierRuvalcaba) [inline components/Boton.jsx]: What is the `{text}` variable use for?

Comment (JavierRuvalcaba) [reply components/Boton.jsx]: The name of the file has a typo.   It should be `Button`

File: hooks/useFetch.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        console.log("te has logeado");
+        } catch (err) {
+     
+        const message= (err.response.data.msg); 
+        throw new Error(message);
+        }
+    };
+    return { loginRequest };
+};
+export default useFetch;
```
Comment (JavierRuvalcaba) [inline hooks/useFetch.js:31]: Maybe, this file will cause merge issues with the one in the @rafael-ramirez-orte 's PR

File: page/Login.jsx [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import {View, Text, Touchable, TouchableOpacity} from 'react-native';
+import Background from './Background';
+import Btn from './Btn';
+import {darkGreen} from './Constants';
+import Field from './Field';
+
```
Comment (JavierRuvalcaba) [inline page/Login.jsx]: Why this file has only import statements?

File: page/LoginPage.jsx [hunk 5]
```diff
@@ -1,18 +1,107 @@
...
+    try {
+      //showLoader();
+      console.log(await loginRequest(data));
+      console.log("te has logeado");
+      navigation.navigate("PatientRegister");
+    } catch (err) {
+      console.log(err);
+    }
+  };
+  return (
```
Comment (JavierRuvalcaba) [inline page/LoginPage.jsx:32]:
```suggestion
  const { loginRequest } = useFetch();
  
  const onSubmit = async (data) => {
    try {
      navigation.navigate("PatientRegister");
    } catch (err) {
      console.log(err);
    }
  };
  
  return (
```

File: page/LoginPage.jsx [hunk 6]
```diff
@@ -1,18 +1,107 @@
...
+          title="Submit"
+          placeholder="Ingresar"
+          onPress={handleSubmit(onSubmit)}
+        />
+      </View>
+    </KeyboardAwareScrollView>
+  );
+};
+const styles = StyleSheet.create({
+  input: {
```
Comment (JavierRuvalcaba) [inline page/LoginPage.jsx:79]:
```suggestion
  );
};

const styles = StyleSheet.create({
  input: {
```

**Review by JavierRuvalcaba — CHANGES_REQUESTED** [review]

**Review by JavierRuvalcaba — APPROVED** [review]

**Review by pedro123-67we — APPROVED** [review]

**Review by pedro123-67we — APPROVED** [review]

**Pull request merged**